### PR TITLE
Converter test

### PR DESCRIPTION
**Pull request by bengaal**
This PR adds a converter test, with faked JSON data.

### CODE REVIEW

File: tests/test_converter.py [hunk 1]
```diff
@@ -23,4 +23,3 @@ def test_conversion():
     marc21_xml = converter.convert_pure_json_to_marc21_xml(
         load_json(join("data", "pure_record_fake.json"))
     )
-    pass
```
Comment (utnapischtim) [inline tests/test_converter.py]: please do such things with a modification of the last commit

File: tests/test_converter.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# Copyright (C) 2021 Graz University of Technology.
+#
+# invenio-records-lom is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Converter tests."""
+
+from invenio_rdm_pure.source.rdm.converter import Converter
+
+def test_conversion(pure_record_merged):
```
Comment (utnapischtim) [inline tests/test_converter.py]: what is this function for?

Comment (bengaal) [reply tests/test_converter.py]: When Pytest runs multiple tests, the `test_` prefix makes them identifiable, so pytest runs them.
https://docs.pytest.org/en/stable/getting-started.html#run-multiple-tests

**Review by utnapischtim — APPROVED** [review]
I saw you created in the next PR (#33) a validation method. I may a good idea to use this method to assert the Converter output, because a string is not yet a valid XML file

**Pull request merged**